lib/resque/tasks.rb

```diff
@@ -41,6 +41,7 @@
   task :preload => :setup do
     if defined?(Rails) && Rails.respond_to?(:application)
       # Rails 3
+      ActiveSupport.run_load_hooks(:before_eager_load, Rails.application)
       Rails.application.eager_load!
```

Comment:
`Rails.application.config.eager_load_namespaces.each(&:eager_load!)` will run eager_load for specified classes/engines.

I suppose will still have to call `ActiveSupport.run_load_hooks(:before_eager_load, Rails.application)` to run the hooks, Or will they execute automatically(internally) by above line(eager_load!)?